contract/test/tribbles-airdrop/e2e.test.js

```diff
@@ -143,7 +143,10 @@ test.serial('deploy contract with core eval: airdrop / airdrop', async t => {
     entryFile: scriptRoots.tribblesAirdrop,
     config: {
       options: {
-        customTerms: makeTerms(),
+        customTerms: {
+          ...makeTerms(),
+          merkleRoot: merkleTreeAPI.generateMerkleRoot(agoricPubkeys),
+        },
```

Comment:
This is headed in the right direction, but the e2e version of [runCoreEval](https://github.com/hindley-milner-systems/dapp-ertp-airdrop/blob/dev/contract/tools/e2e-tools.js#L369) doesn't take a `config` option.

I suggest you codegen the `merkleRoot` at build time using `coreScriptOptions` following the pattern of `swaparoo` in `dapp-agoric-basics`:

```js
  config1({
    name: 'swaparoo',
    permit: swapPermit,
    coreScriptOptions: {
      swaparooCommittee: {
        voterAddresses: env.SWAP_GOV_ADDR ? { v1: env.SWAP_GOV_ADDR } : {},
      },
    },
  }),
```
https://github.com/Agoric/dapp-agoric-basics/blob/3c28a51c2d6f4b4769364b19025665d5a83dc699/contract/rollup.config.mjs#L79-L87

The tweak goes in here...

https://github.com/hindley-milner-systems/dapp-ertp-airdrop/blob/fcb20318a75bb1be4c5c52e2a20fd57c18c34b88/contract/rollup.config.mjs#L74-L77


The core eval script has to have an `options: ` line that matches `new RegExp(`options: Fail.*`, 'g')` like this too:

```
  config = {
    options: Fail`missing options config`,
  },
```
https://github.com/Agoric/dapp-agoric-basics/blob/3c28a51c2d6f4b4769364b19025665d5a83dc699/contract/src/swaparoo.proposal.js#L150-L155

maybe change from:

```js
  const {
    options: Fail`missing options config`,
    // must be supplied by caller or template-replaced
    bundleID = config.bundleID,
  } = config?.options?.[contractName] ?? {};
```

to:

```js
  const config = {
    // must be supplied by caller or template-replaced
    options: Fail`missing options config`,
  };
  const {
    // must be supplied by caller or template-replaced
    bundleID = Fail`missing bundleID`,
    merkleRoot,
  } = options?.[contractName] ?? {};
```

Reply:
The last part of that suggestion wasn't quite right.

We worked it out in office hours, though:

https://github.com/dckc/dapp-ertp-airdrop/tree/dc-config-pattern 4a7fb2a